test/integration/registrar/sign_in_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class RegistrarSignInTest < JavascriptIntegrationTest
```

Comment:
I would name it "RegistrarAreaSignInTest", mainly to avoid name clash (I have experienced such clashes before, and it was time consuming to eventually find out that `setup/teardown` from 2 classes were merged because of the same name of a test class.